README.md

```diff
@@ -297,6 +298,27 @@ non-empty title, it will not be overwritten.
 
 To use this class you will need to have an implicit `Messages` in scope.
 
+#### HmrcPageHeadingLabel and HmrcPageHeadingLegend
+
+These helpers let you use a label or legend as a page heading with a section (caption) displayed above it.
+
+```scala
+@import uk.gov.hmrc.govukfrontend.views.html.components.{GovukInput, Input}
+@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
+@import uk.gov.hmrc.hmrcfrontend.views.config.HmrcPageHeadingLabel
+
+@this(govukInput: GovukInput, pageHeadingLabel: HmrcPageHeadingLabel)
+
+@(myForm: Form[_])(implicit messages: Messages)
+
+@govukInput(
```

Comment:
Should we provide an example for govukRadios too?

Reply:
- [ ] include an example for govukRadios as well